process/smartContract/process.go

```diff
@@ -148,7 +150,7 @@ func NewSmartContractProcessor(args ArgsNewSmartContractProcessor) (*scProcessor
 		return nil, process.ErrNilEpochNotifier
 	}
 
-	apiCosts := args.GasSchedule[core.ElrondAPICost]
+	apiCosts := args.GasSchedule.LatestGasSchedule()[core.ElrondAPICost]
```

Comment:
args.GasSchedule.LatestGasSchedule() should not be check for nil before ?

Reply:
done